tests/testthat/test_ruleLengths.R

```diff
@@ -0,0 +1,58 @@
+test_data6 <- readRDS("testdata/test_rulelength_data/test_e2e_data_6.rds")
+test_data7 <- readRDS("testdata/test_rulelength_data/test_e2e_data_7.rds")
+test_data8 <- readRDS("testdata/test_rulelength_data/test_e2e_data_8.rds")
+test_data9 <- readRDS("testdata/test_rulelength_data/test_e2e_data_9.rds")
+
+test_that("check rule length for run of 6", {
+  
+  results6 <- plot_auto_SPC(test_data6, chartType = "XMR", plotChart = FALSE, runRuleLength = 6)
```

Comment:
Would be good to explicitly pass `periodMin = 21` here, to futureproof these tests against changes to default arguments in `plot_auto_SPC`.